template.html

```diff
@@ -99,7 +99,7 @@
               href="https://twitter.com/share"
               data-size="large"
               data-url="http://www.etcher.io"
-              data-text="Meet Etcher by @resin_io an awesome new way to write SD cards. Give it a shot and you'll never go back!">
+              data-text="Meet Etcher by @resin_io, an awesome new way to write SD cards. Give it a shot and you'll never go back!">
```

Comment:
Could placing the comma here accidentally confuse some twitter clients into thinking it's part of the username? \*shrug\*
(I'm a non-twitter-user myself)